usaspending_api/etl/data_diff_tool.py

```diff
@@ -0,0 +1,42 @@
+from gresearch.spark.diff import *
+
+def data_diff_function(df1, df2):
+    #assumes dataframes as a delta table
+    table1 = spark.read.load(df1)
+    table2 = spark.read.load(df2)
+    return (table1.diff(table2, table1.columns).show)
+
+
+def diff(left: DataFrame, right: DataFrame, unique_key_col="id", compare_cols=[], collect=False):
```

Comment:
Can you add a comment to describe what this method and each of its parameters does? Check out some examples from `usaspending_api/common/helpers/sql_helpers.py`

Reply:
Concur - we use the Google Python Style Guide generally. Take a look at how docstrings are used for functions and methods: https://google.github.io/styleguide/pyguide.html#383-functions-and-methods, esp _`Args`_